ChainSafe/forest
# `auth api-info`

**Summary of changes**
Changes introduced in this pull request:
- Adds `auth api-info` command to forest CLI.
To test: 
- use command `FULLNODE_API_INFO="..." forest auth api-info -p <read/write/admin/sign>`
- copy/paste output as new `FULLNODE_API_INFO` and send another command. Verify that permissions are correct (cannot write with a read token for example)

also bundled in this PR is the fix for `set-default` panicking on a bad address

**Reference issue to close (if applicable)**


Closes #1117 and #1166


**Other information and links**
